modules/sso.php

```diff
@@ -1087,13 +1087,21 @@ public function get_user_data( $user_id ) {
 	 *
 	 * @since 6.9.0
 	 *
-	 * @param array $envelopes Array of JITM messages received after API call.
+	 * @param array  $envelopes    Array of JITM messages received after API call.
+	 * @param string $message_path The message path to ask for.
 	 *
 	 * @return array $envelopes New array of JITM messages. May now contain only one message, about SSO.
 	 */
-	public function inject_sso_jitm( $envelopes ) {
-		// Bail early if that's not the first time the user uses SSO.
-		if ( true != Jetpack_Options::get_option( 'sso_first_login' ) ) {
+	public function inject_sso_jitm( $envelopes, $message_path ) {
```

Comment:
Am I overthinking this, or should we add a default value to the `$message_path` parameter here for backwards compatibility? Just in case there's some other invocation that we've forgotten about, that will only use one argument.

Reply:
d89edb2